beginner_source/translation_transformer.py

```diff
@@ -0,0 +1,399 @@
+"""
+Language Translation with Transformer
+=====================================
+**Author**: `Jayasimha T <https://github.com/iamsimha>`_
+
+This tutorial shows, how to train a translation model from scratch using
+Transformer. We will be using `Multi30k
+dataset <https://github.com/multi30k/dataset>`__ to train a German to
+English translation model.
+
+The data processing and data loading code is taken from an `earlier version
+<https://github.com/pytorch/tutorials/blob/25ece55aac2b3423f8ed06439d0c320bbf40f019/beginner_source/torchtext_translation.py>`__ 
+of tutorial
+
+"""
+
+
+######################################################################
+# Data Processing
+# ---------------
+# 
+# torchtext has utilities for creating datasets that can be easily
+# iterated through for the purposes of creating a language translation
+# model. In this example, we show how to tokenize a raw text sentence,
+# build vocabulary, and numericalize tokens into tensor.
+# 
+# Note: the tokenization in this tutorial requires Spacy We use Spacy
+# because it provides strong support for tokenization in languages other
+# than English. torchtext provides a basic_english tokenizer and supports
+# other tokenizers for English (e.g. Moses) but for language translation -
+# where multiple languages are required - Spacy is your best bet.
```

Comment:
"Spacy is your best bet." I would not to say that spacy is the best bet because there are some other tokenizers for non-english languages. which is being used by the SOTA NLP model.

Reply:
I will remove this.